service/history/api/startworkflow/api.go

```diff
@@ -117,7 +117,16 @@ func NewStarter(
 		return nil, err
 	}
 
+	metricsHandler := workflow.GetPerTaskQueueFamilyScope(
```

Comment:
nit: I'd do lazy init for the metricsHandler if possible to save some gc considering this is a hot path in the system.